### DISCUSSION TITLE
[Question] How to migrate kserve version to newer version

### DISCUSSION THREAD

**Discussion by juhyeon-cha** [kubeflow/kfserving#2522] [Q&A] [answered]
how to migrate kserve(v0.7.0) to latest version?

i only found kfserving to kserve migrate job

**Comment by yuzisun** [comment, marked as answer]
@juhyeon-cha In general we support incremental minor version upgrade path. e.g from 0.7 -> 0.8 -> 0.9. Version jumps are not officially supported, we would recommend reading the release notes and test based on your specific version upgrade path.
